### ISSUE TITLE
vagrant shared folder

### ISSUE THREAD

**Issue by ivoysey** [schmerl/LLStaging#9]
right now the shared folder set up for vagrant mounts to `/vagrant` but everything we run is out of `/home/vagrant`. this means that there's the chance of version skew even inside of the vagrant guest, let alone with respect to the surrounding host environment and whatever git structure exists one level up.

one possibility to fix this is to use sym or hard links within the guest from the shared folder to the home directory. the vagrant docs say that this may or may not work, depending on the OS that's running vagrant, so this might be a bad idea. we don't know much about what the MIT/LL folks are going to do. (https://www.vagrantup.com/docs/synced-folders/basic_usage.html)

we could also just run everything out of `/vagrant` possibly?

**Comment by ivoysey** [comment]
this is kind of something that only effects us, and how irritating it is for us to do dev? although i don't know how LL is going to set up their deployment.

another option would be to have the vagrant script clone from git repos to populate directories. but you wouldn't want that to hit actual real life github, which means you'd want it to know where git repos lived in the host OS FS, which probably means more mucking about with submodules? which does not sound like something that is guaranteed to make this better.

**Comment by schmerl** [comment]
Right. I think this is a dev issue. If we just use /vagrant, then the git workspace gets lots of unnecessary files that need to be filtered out when checking in. And, we might also get clock skew (which I've experienced in my own dev when I've linked to a directory in host from VM.)
